src/components/Table/Body.tsx

```diff
@@ -1,19 +1,20 @@
 import { Tbody, Td, Tr } from '@chakra-ui/react';
 import { Cell, Row, TableBodyProps } from 'react-table';
 import { ColumnInstanceDefinition } from 'model/table/table-definitions';
+import { BaseObject } from 'model/base-object';
 import ActionCell from './ActionCell';
 import ImageCell from './ImageCell';
 
-type BodyProps<T extends object> = {
+type BodyProps<T extends BaseObject> = {
   bodyProps: TableBodyProps;
-  rows: Array<Row<T>>;
+  page: Array<Row<T>>;
   prepareRow: (row: Row<T>) => void;
 };
 
-function Body<T extends object>(props: BodyProps<T>) {
+function Body<T extends BaseObject>(props: BodyProps<T>) {
```

Comment:
Destructure props please.

Reply:
We came to the conclusion (at the meeting on 23.04.2021) that destructing isn't required and necessary due to code duplication (types are declared near `props` in those components).